src/mate/src/Command/StopCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Mate\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Stop all running servers. This will force the AI to restart the server.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+#[AsCommand('stop', 'Stop running servers')]
+class StopCommand extends Command
+{
+    public function __construct(private string $cacheDir)
+    {
+        parent::__construct(self::getDefaultName());
+    }
+
+    public static function getDefaultName(): string
```

Comment:
> symfony/console": "^7.4|^8.0",

Any reason not to use invokable command instead? These static methods are deprecated since 7.3 btw